lbrynet/core/log_support.py

```diff
@@ -147,8 +149,14 @@ def get_loggly_url(token=None, version=None):
     return LOGGLY_URL.format(token=token, tag='lbrynet-' + version)
 
 
+def configure_loggly_handler(*args, **kwargs):
+    if build_type.BUILD == 'dev':
+        return
+    _configure_loggly_handler(*args, **kwargs)
+
+
 @_log_decorator
-def configure_loggly_handler(url=None, **kwargs):
+def _configure_loggly_handler(url=None, **kwargs):
```

Comment:
because this one has the `_log_decorator`, which helper for setting up handlers.  Maybe there are better names I could use for the two.